test/client/js/libs/handlebars/parse.js

```diff
@@ -243,6 +243,18 @@ describe("parse Handlebars helper", () => {
 		name: "monospace",
 		input: "\x11monospace",
 		expected: '<span class="irc-monospace">monospace</span>',
+	}, {
+		name: "reverse with foreground and background colors",
+		input: "\x0313,1fg and bg \x16and reversed",
```

Comment:
I would add tests like `\x0313,01test\x16test\x16test`, `\x0313,01test\x16test\x01,13test` to see what kind of html it generates.